README.md

```diff
@@ -1,20 +1,30 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+-An click handler is a jquery function that allows you to write code that will be executed when the click event occurs
+
+(b) Where in your JS file should you put it, and WHY?
+
+-in `$(document).ready()` so it gets loaded and is waiting to be put to use
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+-You might be using the wrong jQuery reference.
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+
+-Ideally at the bottom, it contains any objects that need to be loaded first. It can go at the top too, though.
 
 ### Question 4
-In an AJAX GET request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent? 
+In an AJAX GET request, what does the argument of the `.done()` callback - in the example below, that would be `data` - represent?
+
+-data will be all the data specified in the URL (all the JSON, in this case)
```

Comment:
Not really. `data` is always the response from the server. For a GET request, that's whatever you requested (e.g. collection of users). For a POST request, the convention is that the server will return whatever you just POSTed (e.g. one single user).